cfgov/v1/wagtail_hooks.py

```diff
@@ -43,6 +43,17 @@ def share_the_page(request, page):
         flush_akamai(page)
 
 
+@hooks.register('after_delete_page')
+def log_page_deletion(request, page):
+    logger.warning(
+        'User {user} with ID {user_id} deleted page {title} with ID {page_id}'.format(
```

Comment:
This will raise a `UnicodeEncodeError` if the title or username have any unicode characters in them. To reproduce locally, create a page named "Unicodë", then go to that page's `/delete` page and try deleting. I believe a fix is to just use a string like `u'User {user} with...'` (note the 'u' prefix).